tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -243,7 +243,7 @@ def fake_magpie_api_request(url, data):
 def test_harvesting_arxiv_workflow_rejected(
     mocked_api_request_beard_block, mocked_api_request_magpie,
     mocked_api_request_beard, mocked_download,
-    app, record_oai_arxiv_plots):
+    small_app, record_oai_arxiv_plots):
```

Comment:
I'll write it here, because I don't know where to write this: use `small_app` whenever you need all the components of the application (DB, ES, Celery...) to be there, but you don't need the records. Use `app` when you need all the demo records to be loaded.